now.json

```diff
@@ -445,6 +445,10 @@
       "source": "/guides/nextjs(.*)",
       "destination": "/guides/tina-cloud/add-tinacms-to-existing-site/overview/"
     },
+    {
+      "source": "/guides/tinacms(.*)",
+      "destination": "/guides/"
+    },
```

Comment:
This was resurrected.  I assume we also want to _keep_ the ultimate redirect if everything else falls through?
```js
    {
      "source": "/guides/(.*)",
      "destination": "/docs/setup-overview/"
    },
```

Reply:
Just removed that global redirect It was actually redirect any guide link, (including the new one), back to the docs root